src/LodeRunner.API.Test/IntegrationTests/Extensions/EqualException.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This class is the same as ContainsException. Why is it needed?

Reply:
They are not, one derives from Xunit.Sdk.EqualException and the other one derives from Xunit.Sdk.EqualException, so the correct exception type has been thrown,